generators/plopGenerateModule.js

```diff
@@ -0,0 +1,28 @@
+﻿module.exports = function (plop) {
+  plop.setGenerator('component', {
```

Comment:
```suggestion
  plop.setGenerator('module', {
```
Acredito que seja `module` o nome do generator, certo?